src/amo/pages/Home/index.js

```diff
@@ -43,18 +37,6 @@ export const FEATURED_COLLECTIONS = [
   { slug: 'password-managers', userId: MOZILLA_USER_ID },
 ];
 
-export const isFeaturedCollection = (
```

Comment:
This function was also used in `amo/pages/Collection/index.js`, but I don't think it was actually needed there, so its usage from that file can be removed as well. Without doing that we get test failures.